collection/prem.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "prem",
+    "meaning": "love",
+    "aliases": [
+        "sneh"
+    ],
+    "translations": {
```

Comment:
Can you please update the translations to use the correct ISO 639-3 code instead of the language names?
List of ISO codes can be found [here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).